### PR TITLE
Added CustomHttpResponseHeadersFilter

### PR DESCRIPTION
**Pull request by AdiWehrli**
With this filter we can configurably define response headers to be added to each HTTP response.

### CODE REVIEW

File: src/main/java/org/akhq/middlewares/CustomHttpResponseHeadersFilter.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import javax.inject.Inject;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Filter("/**")
+public class CustomHttpResponseHeadersFilter implements HttpServerFilter {
+
+    private static final Logger LOG = LoggerFactory.getLogger(CustomHttpResponseHeadersFilter.class);
+    public static final String PROPERTY_CUSTOM_RESPONSE_HEADERS = "akhq.http.headers.custom-response-headers";
```
Comment (tchiotludo) [inline src/main/java/org/akhq/middlewares/CustomHttpResponseHeadersFilter.java:24]:
```suggestion
    public static final Map<String, String> PROPERTY_CUSTOM_RESPONSE_HEADERS = "akhq.server.custom-response-headers";
```
Let users provide the headers as Map directly and change the variable name

**Review by tchiotludo — CHANGES_REQUESTED** [review]
Almost good 

Please : 
- Add some docs [here](https://github.com/tchiotludo/akhq/blob/dev/docs/docs/configuration/akhq.md)
- Add [some examples here](https://github.com/tchiotludo/akhq/blob/dev/application.example.yml)

File: src/main/java/org/akhq/middlewares/CustomHttpResponseHeadersFilter.java (hunk 1 above)
Comment (AdiWehrli) [reply src/main/java/org/akhq/middlewares/CustomHttpResponseHeadersFilter.java:24]: Well that didn't work in my unit tests.

Comment (tchiotludo) [reply src/main/java/org/akhq/middlewares/CustomHttpResponseHeadersFilter.java]: Maybe you need to create a true configuration for that one like this [one](https://github.com/tchiotludo/akhq/blob/dev/src/main/java/org/akhq/configs/Connection.java#L30)

Comment (AdiWehrli) [reply src/main/java/org/akhq/middlewares/CustomHttpResponseHeadersFilter.java]: Well, we cannot assign a String to a Map as you suggested: ```java
public static final Map<String, String> PROPERTY_CUSTOM_RESPONSE_HEADERS = "akhq.server.custom-response-headers";
```

Comment (tchiotludo) [reply src/main/java/org/akhq/middlewares/CustomHttpResponseHeadersFilter.java]: Yes I know I want people to fill it like that: 

```yaml 

akhq:
  server:
    custom-response-headers:
      customheaders: values
      customheaders2: values2

```

Comment (tchiotludo) [reply src/main/java/org/akhq/middlewares/CustomHttpResponseHeadersFilter.java]: good catch :+1: 
```yaml 
akhq:
  server:
    custom-response-headers:
     - key: "custom-header-1"
       value: "values1a values1b"
```

Comment (AdiWehrli) [reply src/main/java/org/akhq/middlewares/CustomHttpResponseHeadersFilter.java]: Configuration is now in class org.akhq.configs.Server. Example:
```yaml
akhq:
  server:
    customHttpResponseHeaders:
      - name: "Content-Security-Policy"
        value: "default-src 'none'; frame-src 'self'; script-src 'self'; connect-src 'self'; img-src 'self'; style-src 'self'; frame-ancestors 'self'; form-action 'self'; upgrade-insecure-requests"
      - name: "X-Permitted-Cross-Domain-Policies"
        value: "none"
```

**Comment by AdiWehrli** [comment]
What exactly does this error message mean: `Couldn't find any clusters on your configuration file, please ensure that the configuration file is loaded correctly`?

**Comment by AdiWehrli** [comment]
Finally all checks have passed :)

**Comment by AdiWehrli** [comment]
I added another test for the situation if no custom HTTP response headers were configured.

**Comment by AdiWehrli** [comment]
So, I will not change anything at this moment.

**Comment by tchiotludo** [comment]
merged, thanks a lot :+1:

**Comment by tchiotludo** [comment]
It's available on [dev](https://akhq.io/docs/dev.html) version.
The release will be when [this](https://github.com/tchiotludo/akhq/issues/1215) was done (upgrade to latest micronaut version)

**Comment by tchiotludo** [comment]
Still better to ask for PR instead on building on your own house :smiley_cat:

**Pull request merged**